spring-rabbit/src/test/java/org/springframework/amqp/rabbit/test/RabbitMatchers.java

```diff
@@ -39,7 +39,7 @@ private RabbitMatchers() {
 
 		private final String pattern;
 
-		private RegexMatcher(String pattern) {
+		RegexMatcher(String pattern) {
```

Comment:
No need - we never instantiate `RabbitMatchers` - it has a private ctor because it's just a utility class with static methods.